--- tensorboard-1.13.0/tensorboard/pip_package/build_pip_package.sh~	2019-02-25 18:44:48.000000000 +0000
+++ tensorboard-1.13.0/tensorboard/pip_package/build_pip_package.sh	2019-02-27 03:28:57.874078221 +0000
@@ -110,7 +110,7 @@
 
 # Require wheel for bdist_wheel command, and setuptools 36.2.0+ so that
 # env markers are handled (https://github.com/pypa/setuptools/pull/1081)
-pip install -qU wheel 'setuptools>=36.2.0'
+#pip install -qU wheel 'setuptools>=36.2.0'
 
 python setup.py bdist_wheel --python-tag py2 >/dev/null
 python setup.py bdist_wheel --python-tag py3 >/dev/null
@@ -162,12 +162,12 @@
 
 case "${smoke}" in
   "all")
-    smoke 2 "${tf_version}"
-    smoke 3 "${tf_version}"
+#    smoke 2 "${tf_version}"
+#    smoke 3 "${tf_version}"
     ;;
   "1")
     # Empty string indicates to use the default "python".
-    smoke "" "${tf_version}"
+#    smoke "" "${tf_version}"
     ;;
   "0")
     printf "\nSkipping smoke test\n\n"
